dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GeneralIT.java

```diff
@@ -711,6 +711,17 @@ public void testVersionTagHide() {
         final long count2 = testingPostgres
             .runSelectStatement("select count(*) from tag t, version_metadata vm where vm.hidden = 't' and t.id = vm.id", long.class);
         assertEquals("there should be 0 hidden tag", 0, count2);
+
+        tags = new ArrayList<>();
+        updatedTag = tag.get();
+        updatedTag.setHidden(true);
+        updatedTag.setVersionMetadata(null);
```

Comment:
Oh, I see the reason for Gary's comment. We should not be able to blank the version data by API this way. 
Set just the hidden attribute to avoid breakage when we fix this

Reply:
@garyluu Yeah I misunderstood, I also agree we should not be able to blank the version metadata directly.

Reply:
Reverted the integration test to what it was on develop. No need to add new tests as current tests already set the hidden attribute.